tools/DoctorMyPass.py

```
@@ -0,0 +1,80 @@
+            f.write(f"Machine: {info['machine']}\n")
+            f.write(f"Processor: {info['processor']}\n")
+            f.write(f"Architecture: {info['architecture']}\n")
+            f.write(f"Current Working Directory: {info['cwd']}\n")
+            f.write("\nsys.path:\n")
+            for p in info['sys_path']:
+                f.write(f" - {p}\n")
+            f.write("\nImportable Modules:\n")
+            for mod in info['modules']:
+                f.write(f" - {mod}\n")
+        print(f"\U0001F4DD Diagnostic saved to {filename}")
+    except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)